### PR TITLE
Reorganize configs

### PR DESCRIPTION
**Pull request by AlexejPenner**
As we have a two stepped system

Code goes through dev-staging-prod

-> only models trained from prod code gets the additional staging->prod flow in the mcp , thus i have separated out the models

### CODE REVIEW

**Comment by strickvl** [comment]
image

File: llm-complete-guide/configs/production/embeddings.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+# environment configuration
+settings:
+  docker:
+    parent_image: "zenmldocker/prepare-release:base-0.68.1"
+    requirements:
+      - langchain-community
+      - ratelimit
+      - langchain>=0.0.325
+      - langchain-openai
```
Comment (strickvl) [inline llm-complete-guide/configs/production/embeddings.yaml:11]:
```suggestion
      - ratelimit
```

Not sure how these dependencies ended up here, but we don't need / want them.

File: llm-complete-guide/configs/production/synthetic.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# environment configuration
+settings:
+  docker:
+    requirements:
+      - langchain-community
+      - ratelimit
+      - langchain>=0.0.325
+      - langchain-openai
```
Comment (strickvl) [inline llm-complete-guide/configs/production/synthetic.yaml:8]:
```suggestion
      - ratelimit
```

Again, no langchain needed

Comment (strickvl) [reply llm-complete-guide/configs/production/synthetic.yaml:8]: you might need to add `openai`, though...

File: llm-complete-guide/configs/staging/embeddings.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+# environment configuration
+settings:
+  docker:
+    parent_image: "zenmldocker/prepare-release:base-0.68.0"
+    requirements:
+      - langchain-community
+      - ratelimit
+      - langchain>=0.0.325
+      - langchain-openai
```
Comment (strickvl) [inline llm-complete-guide/configs/staging/embeddings.yaml:11]:
```suggestion
      - ratelimit
```

Possibly adding openai

File: llm-complete-guide/configs/production/embeddings.yaml (hunk 1 above)
Comment (strickvl) [reply llm-complete-guide/configs/production/embeddings.yaml:11]: might need to add openai

File: llm-complete-guide/configs/staging/synthetic.yaml [hunk 4]
```diff
@@ -0,0 +1,39 @@
+# environment configuration
+settings:
+  docker:
+    requirements:
+      - langchain-community
+      - ratelimit
+      - langchain>=0.0.325
+      - langchain-openai
```
Comment (strickvl) [inline llm-complete-guide/configs/staging/synthetic.yaml:8]:
```suggestion
      - ratelimit
```

and adding openai I think

**Comment by AlexejPenner** [comment]
> image

that model reallx struggled with the text, this is great though

**Pull request merged**